src/yb/yql/pgwrapper/pg_wrapper.cc

```diff
@@ -353,9 +353,9 @@ Status PgWrapper::Start() {
   // users change the shared_preload_libraries conf parameter.
   if (FLAGS_pg_stat_statements_enabled) {
     argv.push_back("shared_preload_libraries=pg_stat_statements,yb_pg_metrics,pgaudit,"
-      "pg_hint_plan");
+      "pg_hint_plan,postgres_fdw");
```

Comment:
This should use the method from: https://github.com/yugabyte/yugabyte-db/pull/9576.